package.json

```diff
@@ -244,6 +245,18 @@
     "trailingComma": "es5",
     "printWidth": 100
   },
+  "remarkConfig": {
+    "settings": {
+      "commonmark": true,
+      "fences": true,
+      "listItemIndent": "1",
+      "rule": "-",
+      "ruleSpaces": false
+    },
```

Comment:
These settings were chosen to reduce the size of the initial diff. These settings are listed under [`remark-stringify`](https://github.com/remarkjs/remark/tree/master/packages/remark-stringify#options)